pkg/draft/context.go

```diff
@@ -48,8 +48,8 @@ func newAppContext(s *Server, req *rpc.UpRequest, out io.Writer) (*AppContext, e
 
 	// inject certain values into the chart such as the registry location,
 	// the application name, and the application version.
-	tplstr := "image.name=%s,image.org=%s,image.registry=%s,image.tag=%s,basedomain=%s,ondraft=true"
-	inject := fmt.Sprintf(tplstr, req.AppName, s.cfg.Registry.Org, s.cfg.Registry.URL, imgtag, s.cfg.Basedomain)
+	tplstr := "image.name=%s,image.org=%s,image.registry=%s,image.tag=%s,basedomain=%s,ondraft=true,draft=%s"
```

Comment:
Would it be better to reference `local.DraftLabelKey` here (and make `local.draftLabelKey` public as `local.DraftLabelKey`), or do we not want third party libraries to use that package?